### ISSUE TITLE
2018.08-rc3 fails to execute berryboot-script

### ISSUE THREAD

**Issue by macmpi** [maxnet/berryboot#528]
Hi,
Just started playing with 2018.08-rc3 with images containing (previously working) `berryboot-init` scripts.
[init execution fails](https://github.com/maxnet/berryboot/blob/berryboot2018/external/package/berrybootgui2/init#L100) getting into kernel panic with:
```
Mounting image myimage.img...
Executing init script
/init: .: line 100: berryboot-init: not found
```
Curious as init does not seem to be changed from former versions.
But that release has many other changes, so it's hard to follow.
Thanks for any hint.

EDIT: note I moved from earlier version with a manual update (as I always did): replacing all files (except `config.txt` and `cmdline.txt`) on SD, and executing `gzip -dc /boot/shared.tgz | tar x -C /mnt/shared` under console.
Maybe this update mechanism does not apply for such a major upgrade?

**Comment by maxnet** [comment]
Can you try if it works better with attached berryboot.img?

[berryboot.zip](https://github.com/maxnet/berryboot/files/2364348/berryboot.zip)

**Comment by macmpi** [comment]
Thanks for the very quick turnaround.
Indeed the supplied image passes fine.
However, I fear the same sort of issues may arise into existing scripts and so on...
Has busybox `ash` some sort of settings/options/environment variables that can be set to match previous environment, so that we can expect same workflow as previously?

Note: while it passes, the image got into weird startup problems & hangs (EXT4-fs errors): maybe the original kernel panic just trashed my original image environment :(

**Comment by maxnet** [comment]
No idea.
Busybox does have configuration options, if you have the Berryboot source code:

```
cd buildroot*
make busybox-menuconfig
```

I have a plane to catch later tonight. So cannot look into it right now.

**Comment by symbios24** [comment]
Hi i noticed that there is a new berryboot version (berryboot-20180909-pi2-pi3.zip) but i cannot find the change-log anywhere,does it fix the kernel panic message that happens sometimes in my case in the berryboot image of retropie when i shutdown the os?

Thanks

**Comment by macmpi** [comment]
hopefully 2018.08 final, Released September 6th, fixes it with some `busybox` fixes (not checked details)

**Comment by maxnet** [comment]
They disabled the functionality by default in January: https://git.busybox.net/busybox/commit/shell/ash.c?id=01f7b9e182e257c12f7a7dec76c6b4168605ecd2
Turned it back on.
